src/App/ImportUshahidiV2/Repositories/ImportRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Ushahidi Export Batch Repository
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2018 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\ImportUshahidiV2\Repositories;
+
+use Ushahidi\App\ImportUshahidiV2\Import;
+use Ushahidi\App\ImportUshahidiV2\Contracts\ImportRepository as ImportRepositoryContract;
+
+class ImportRepository /*extends EloquentRepository*/ implements ImportRepositoryContract
+{
+
+    public function create(Import $model) : int
+    {
+        return $model->save() ? $model->id : false;
```

Comment:
This is slightly pointless. I could just use the model directly. For now it allows for swapping out Eloquent later though

Reply:
_shrugs_
That's fine I think